ci-operator/step-registry/openshift-power-monitoring/install/kepler/OWNERS

```diff
@@ -0,0 +1,5 @@
+approvers:
+- vprashar2929
```

Comment:
Are you the only one?

Reply:
Currently yes. But might change in future